heatexchanger/sol.txt

```diff
@@ -1,132 +1,132 @@
 
 Cost
 ----
- 0.0003311 [N/W] 
+ 0.003753 [1/W] 
 
 Free Variables
 --------------
         | Layer
-  D_air : 0.02093                                         [N]        total air drag
-D_water : 0.01812                                         [N]        total water drag
-      Q : 117.9                                           [W]        heat transferred from air to liquid
- V_mtrl : 79.02                                           [cm**3]    volume of material
-  V_tot : 100                                             [cm**3]    total volume
+  D_air : 0.1                                             [N]        total air drag
+      Q : 266.4                                           [W]        heat transferred from air to liquid
+      V : 100                                             [cm**3]    total volume
+ V_mtrl : 15.33                                           [cm**3]    volume of material
```

Comment:
it's a bit unintuitive that the low-drag one is the one that's 79% material...we seem in general to not be very sensitive to material quantity?